cmd/statusd/main.go

```diff
@@ -149,6 +151,7 @@ func main() {
 	// Run stats server.
 	if *metrics {
 		go startCollectingNodeMetrics(interruptCh, backend.StatusNode())
+		gethmetrics.CollectProcessMetrics(3 * time.Second)
```

Comment:
Isn't this call blocking? In geth it's called in a goroutine: [`go metrics.CollectProcessMetrics(3 * time.Second)`](https://sourcegraph.com/github.com/status-im/status-go@4154af136d60020ee441619f72fa1032b1cb8b59/-/blob/vendor/github.com/ethereum/go-ethereum/cmd/geth/main.go#L212:14).